src/Common/src/Interop/Interop.Globalization.Extensions.manual.cs

```diff
@@ -57,6 +58,10 @@ internal partial class mincore
                                         [System.Runtime.InteropServices.OutAttribute()]
                                         char[] destenation,
                                         int destenationLength);
+
+
+        internal const int IDN_ALLOW_UNASSIGNED = 0x1;
+        internal const int IDN_USE_STD3_ASCII_RULES = 0x2;
```

Comment:
I assume you are going to factoring these using the new interop guidance at some other point?